### PR TITLE
Remove React Vite Boilerplate 

### PR DESCRIPTION
**Pull request by johnsonav1992**
- Got rid of the ol' crapola

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,34 +1,6 @@
...
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <div>Sup brotha</div>
```
Comment (eliotmjohnson) [inline src/App.tsx]: Don't use generic elements like div.

**Review by eliotmjohnson — CHANGES_REQUESTED** [review]
Hey check out my notes and make changes plz.

File: src/App.tsx (hunk 1 above)
Comment (johnsonav1992) [reply src/App.tsx]: Kkkkkkk

**Review by eliotmjohnson — APPROVED** [review]
Looks swell

**Pull request merged**